database/migrations/1669317722107_reservations.ts

```diff
@@ -0,0 +1,23 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class extends BaseSchema {
+  protected tableName = 'reservations'
+
+  public async up () {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments('id')
+      table.date('date').notNullable()
+      table.integer('hour').notNullable()
```

Comment:
table.dateTime('hour', { useTz: true }).defaultTo(this.now())

Reply:
![Imagen desde la web](https://cdn-3.expansion.mx/dims4/default/ff7a8b3/2147483647/strip/true/crop/500x381+0+0/resize/1200x914!/quality/90/?url=https%3A%2F%2Fcdn-3.expansion.mx%2F1b%2F65%2F2a5166a640d48403e696b714a28d%2Fexito-meme.jpg)